### PR TITLE
fix: error ordering for traces

### PR DESCRIPTION
**Pull request by thisisnithin**
## Motivation and Context

Show error count and rate serparately and order by correctly.

## Checklist

- [ ] I have discussed my proposed changes in an issue and have received approval to proceed.
- [x] I have followed the coding standards of the project.
- [ ] Tests or benchmarks have been added or updated.
- [ ] Documentation has been updated on [https://github.com/wundergraph/cosmo-docs](https://github.com/wundergraph/cosmo-docs).
- [ ] I have read the [Contributors Guide](https://github.com/wundergraph/cosmo/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: controlplane/src/core/repositories/analytics/AnalyticsRequestViewRepository.ts [hunk 1]
```diff
@@ -112,8 +112,12 @@ export class AnalyticsRequestViewRepository {
       unit: Unit.Nanoseconds,
       title: 'P95 Latency',
     },
-    errorsWithRate: {
-      title: 'Errors (Rate%)',
+    errors: {
+      title: 'Errors',
+    },
+    errorRate: {
+      title: 'Error Rate%',
```
Comment (StarpTech) [inline controlplane/src/core/repositories/analytics/AnalyticsRequestViewRepository.ts:119]: `Error Rate %` or `Error Rate in %` but `Rate%` looks a like typo 😄

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: controlplane/src/core/repositories/analytics/AnalyticsRequestViewRepository.ts (hunk 1 above)
Comment (thisisnithin) [reply controlplane/src/core/repositories/analytics/AnalyticsRequestViewRepository.ts]: Yeah typo, its supposed to be just `Error Rate`

**Comment by thisisnithin** [comment]
> Could you elaborate a bit on what was wrong and how you fixed it?

We were querying string instead of numeric values which leads to incorrect sorting. I now simplify it into 2 columns which are both numeric

**Review by StarpTech — APPROVED** [review]
Lgtm

**Pull request merged**